README.md

```diff
@@ -1,49 +1,47 @@
 # L[ightningPayJ]oin: Automated External Lightning Channel Funding
 
-This server leverages Pay-to-Endpoint to negotiate channel funding on your lightning node from an external wallet supporting BIP78. Previously, external funding could only be done by [signing PSBTs manually](https://github.com/lightningnetwork/lnd/blob/master/docs/psbt.md). A node running this software can open a new scheduled lightning channel upon payment receipt from e.g. BTCPayServer with a normal QR scan transfer flow.
+Loin leverages Pay-to-Endpoint (payjoin) to negotiate a channel open for your lightning node, initiated by any BIP78 supporting wallet. Previously, custom PSBT channel funding could only be done by [signing PSBTs manually](https://github.com/lightningnetwork/lnd/blob/master/docs/psbt.md). A node running Loin can open a new lightning channel for every received payment, provided the payer supports bip78. E.g. BTCPayServer or Blue wallet, following the normal QR scan transfer flow. Read the article about [Lightning Powered PayJoin](https://chaincase.app/words/lightning-payjoin?ref=github) to hear all the benefits!
 
 ## ⚠️ WARNING: ULTRA-EXPERIMENTAL SOFTWARE
 
 ❗️ **This project is pre-alpha quality and lacks basic checks. It is NOT fully BIP78-compliant** ❗️
 
-Any unexpecected node setup or message will likely cause permanent loss of funds.
+Any unexpecected node version will likely cause permanent loss of funds.
 
 For the love of god, do NOT use it on mainnet just yet!
 If you don't want to wait long, help with reviews and PRs
 
-## Benefits
+### Expected fee savings
 
-Read the article about [Lightning Powered PayJoin](https://chaincase.app/words/lightning-payjoin?ref=github).
+In the traditional path for opening a lightning channel, you first must fund the LND wallet and then make a second transaction to fund the opening of the channel, a total of two transactions.
+Loin however does this all in a single transaction, saving 106 vB (68 vB input + 22 vB script pubkey + 8 vB output amount + 4 vB version + 4vB timelock).
 
-### Expected fee savings
+If you wanted to fund a number of channels, say 10 channels, this would traditionally take 11 transactions. Loin can do this in a single transaction - saving 1060 vB.
```

Comment:
```suggestion
If you wanted to fund a number of channels, say 10 channels, this would traditionally take 11 transactions. Nolooking can do this in a single transaction saving 1060 vB.
```